airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -385,6 +385,18 @@
         "description": "Number of file buffers allocated for writing data. Increasing this number is beneficial for connections using Change Data Capture (CDC) and up to the number of streams within a connection. Increasing the number of file buffers past the maximum number of streams has deteriorating effects",
         "examples": ["10"],
         "order": 9
+      },
+      "use_1s1t_format": {
+        "type": "boolean",
+        "description": "(Beta) Use <a href=\"https://github.com/airbytehq/airbyte/issues/26028\" target=\"_blank\">Destinations V2</a>. Contact Airbyte Support to participate in the beta program.",
+        "title": "Use Destinations V2 (Early Access)",
+        "order": 10
+      },
+      "raw_data_schema": {
+        "type": "string",
+        "description": "(Beta) The schema to write raw tables into",
+        "title": "Destinations V2 Raw Table Schema (Early Access)",
+        "order": 11
```

Comment:
cc @Hesperide - we are going right into early-access for all users, no private beta thi time around